docs/index.rst

```diff
@@ -7,8 +7,30 @@
     :hidden:
     :caption: Getting Started
 
-    get_start/index.rst
-    user_guide/index.rst
+    tutorials/overview.rst
```

Comment:
your adding in an old version of this file

Reply:
Do you mind clarifying here?

Reply:
Corrected, are we good to merge?